webpack.config.js

```diff
@@ -6,35 +6,53 @@
 // update, you can easily restore the functionality you've added.
 // -----------------------------------------------------------------------------
 
```

Comment:
The changes to `webpack.config.js` were taken from the [upstream `webpack.config.js` template](https://github.com/laurent22/joplin/blob/dev/packages/generator-joplin/generators/app/templates/webpack.config.js). This allows CodeMirror 6 packages to be `import`ed by `codeMirror6.ts`.